src/routes/JobViewRoute.js

```diff
@@ -64,31 +69,42 @@ class JobViewRoute extends React.Component {
 
   render() {
     const { resources } = this.props;
-    const job = get(resources, 'job.records[0]', {});
-    const name = get(job, 'name', '');
+    const job = resources?.job?.records?.[0] ?? {};
+    const name = job?.name ?? '';
+    const jobClass = job?.class ?? '';
+
+    let deleteMessageId = 'ui-local-kb-admin.job.delete.message';
+    let deleteHeadingId = 'ui-local-kb-admin.job.delete.heading';
+
+    if (jobClass !== '') {
+      deleteMessageId = deleteMessageId.concat(`.${jobClass}`);
```

Comment:
can you change this to `${deleteMessageId}.${jobClass}` and similarly with the
 `deleteHeadingId = `${deleteHeadingId}.${jobClass}`

Reply:
Ohhh, that's way cleaner, cheers Adi